js/coinsbit.js

```diff
@@ -0,0 +1,1114 @@
+            //              market: 'ETH_USDT',
+            //              makerFee: '0.002',
+            //              dealFee: '0.01597712',
+            //              dealStock: '0.002',
+            //              dealMoney: '7.98856357',
+            //              marketName: 'ETH_USDT'
+            //          },
+            //          ...
+            //        ]
+            //     },
+            //     code: '200',
+            // }
+            const data = this.safeValue (response, 'result', {});
+            const records = this.safeValue (data, 'records', []);
+            return this.parseOrders (records, undefined, since, limit, params);
+        }
+    }
+
+    parseOrder (order, market = undefined) {
+        // createOrder & cancelOrder  ( & fetchOpenOrders, but 'id' instead of 'orderId')
+        //     {
+        //       orderId: '8629354782',
+        //       market: 'ETH_USDT',
+        //       price: '4000',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640212275.928',
+        //       dealMoney: '7.98856357',
+        //       dealStock: '0.002',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0',
+        //       dealFee: '0'
+        //     }
+        //
+        //  fetchClosedOrder
+        //      {
+        //         dealOrderId: '8631729321',
+        //         id: '1595921',
+        //         time: '1640258718.402',
+        //         fee: '0.01572657',
+        //         price: '3931.64271609',
+        //         amount: '0.002',
+        //         role: '2',
+        //         deal: '7.86328543'
+        //      }
+        //
+        //  fetchClosedOrders
+        //      {
+        //         amount: '0.002',
+        //         price: '4000',
+        //         type: 'limit',
+        //         id: '8629354787',
+        //         side: 'buy',
+        //         ctime: '1640212275.928',
+        //         takerFee: '0.002',
+        //         ftime: '1640212275.93',
+        //         market: 'ETH_USDT',
+        //         makerFee: '0.002',
+        //         dealFee: '0.01597712',
+        //         dealStock: '0.002',
+        //         dealMoney: '7.98856357',
+        //         marketName: 'ETH_USDT'
+        //      },
+        const isFromFCO = 'role' in order;
+        const isFromFCOs = 'ctime' in order;
+        let id = undefined;
+        let tradeId = undefined;
+        if (isFromFCOs) {
+            id = this.safeString (order, 'id');
+        } else if (isFromFCO) {
+            id = this.safeString (order, 'dealOrderId');
+            tradeId = this.safeString (order, 'id'); // <<<TODO>>> this 'TRADE-ID' information is only in fetchClosedOrder, so is this correct implementation? or it should be sent to 'parseTrade' (as its structure is more like of 'parseTrade' endpoints)
```

Comment:
From the response of fetchClosedOrder it looks like they actually return trades from that endpoint, cause an order cannot have a `dealOrderId`. I think the implementation of fetchClosedOrder and hence the parsing code here may be wrong.

Reply:
Can you mention your thought :

`account/order`
and
`account/trades`

have very identical  endpoint  responses ( https://www.notion.so/coinsbitwsapi/API-COINSBIT-WS-API-COINSBIT-cf1044cff30646d49a0bab0e28f27a87 ), the only difference is that the latter returns the array (as expected for `fetchOrderTrades`) , however first one - which ccxt unified method does it relate to?

Reply:
If possible, please test, whether they allow calling account/trades without an orderId. We will decide depending on that. If they do, then `account/order` = fetchOrderTrades, and `account/trades` = fetchMyTrades. If they always require an order id in both endpoints – we should simply toss a coin.

Reply:
I think I still have a problem (after your last above comment) to understand those endpoints again. Let me mention:
- they have requirement of `orderId`  (otherwise they return error, like `..The order id field is required..`).
So, in that case, as you say, they should be:
`account/order` = fetchOrderTrades
`account/trades` = fetchMyTrades

However, the problem is that:
- their `account/order` doesn't return the array of objects, instead it returns one object (so, how it would be `fetchOrderTrades`?
- `account/trades` returns array, and  it shouldn't be `fetchMyTrades`, but `fetchOrderTrades`.  However, what about `account/order` endpoint? 

(in discord, I incorrectly described what I wanted to say with fetchMyTrades, however, that method is out of scope in this issue, so I'll not mention that atm).